### ISSUE TITLE
pop-up window

### ISSUE THREAD

**Issue by rcyggdra** [matyalatte/tuw#34]
Because the command window does not support Chinese character output, 
![PixPin_2024-06-30_19-15-24](https://github.com/matyalatte/tuw/assets/8652464/d7aa58a2-01b7-49a3-a80b-c585931e2cf6)
I use `start cmd /k` to run the command in a new CMD window. I want to cancel the pop-up window after running tuw. What should I do?
![PixPin_2024-06-30_19-19-02](https://github.com/matyalatte/tuw/assets/8652464/1be7f37a-08a0-4864-8389-8e6c8cbf1eee)

**Comment by matyalatte** [comment]
Thanks for the report.

> Because the command window does not support Chinese character output

You can see it supports Chinese characters already.
![output](https://github.com/matyalatte/tuw/assets/69258547/d6fd85ee-97dc-4bd9-9e50-0b8a071e4bc8)

Probably, the cause is locale (like gb2312 and big5) for python. Tuw only supports user's default locale. So, it can show garbled strings if your python script changes the setting.
I opened a new issue for it.
[Sync locale with child processes on Windows · Issue #36 · matyalatte/tuw](https://github.com/matyalatte/tuw/issues/36)

> I use start cmd /k to run the command in a new CMD window. I want to cancel the pop-up window after running tuw. What should I do?

I opened another issue because there is nothing you can do for now.
[Option to disable message dialogues · Issue #37 · matyalatte/tuw](https://github.com/matyalatte/tuw/issues/37)

**Comment by matyalatte** [comment]
I'll close this issue because I added an option to skip the pop-up window at [v0.7.0](https://github.com/matyalatte/tuw/releases/tag/v0.7.0).

You can skip the dialog with `"show_success_dialog": false`.

```json
{
    "gui": [
        {
            "label": "Option to skip success dialog",
            "show_success_dialog": false,
            "command": "echo Hello!",
            "components": []
        }
    ]
}
```

The garbled string issue will be addressed in #36.
